js/import/rules.import.js

```diff
@@ -12,18 +12,18 @@
 import xPathToCss from '../libs/vendors/xpath-to-css/xpath-to-css.js';
 
 const XPATH_BODY = '/html[1]/body[1]';
+const MAPPING_CONFIG_KEYS = ['name', 'value'];
 
 const baseTransformRules = {
   cleanup: {
     start: [],
-    end: [],
   },
   blocks: [
     {
       type: 'metadata',
       insertMode: 'append',
       params: {
-        metadata: {},
+        cells: {},
```

Comment:
A `metadata` specific param is no longer needed. All block customizations can be handled by the same `cells` property which can either be a name/value pair object or a two dimensional array of selectors.